images/router/haproxy/conf/haproxy-config.template

```diff
@@ -155,8 +175,13 @@ backend be_edge_http_{{$cfgIdx}}
   mode http
   balance leastconn
   timeout check 5000ms
+  {{ if (eq $cfg.TLSTermination "") }}
+  cookie OPENSHIFT_{{$cfgIdx}}_SERVERID insert indirect nocache httponly
+  {{ else }}
+  cookie OPENSHIFT_EDGE_{{$cfgIdx}}_SERVERID insert indirect nocache httponly secure
+  {{ end }}
```

Comment:
Are individual developers in an enterprise going to be making that decision?  Or the admin?  If we don't *have* to support it per app then we shouldn't until we have a concrete use case for it.